infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -89,6 +89,8 @@
     /** The node Id, stored as a base64 string. */
     private static final String NODE_ID = "NODE_ID";
 
+    public static final int RECORDS_PER_LOG_FILE = 10_000;
```

Comment:
The way the LogUnit partitions its data is implementation detail and its not meant to be exposed.

Reply:
Will make it another configuration parameter.